tests/components/wemo/conftest.py

```diff
@@ -0,0 +1,81 @@
+"""Fixtures for pywemo."""
+import asynctest
+import pytest
+import pywemo
+
+from homeassistant.components.wemo import CONF_DISCOVERY, CONF_STATIC
+from homeassistant.components.wemo.const import DOMAIN
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import patch
+
+MOCK_HOST = "127.0.0.1"
+MOCK_PORT = 50000
+MOCK_NAME = "WemoDeviceName"
+MOCK_SERIAL_NUMBER = "WemoSerialNumber"
+
+
+@pytest.fixture(name="pywemo_model")
+def pywemo_model_fixture():
+    """Fixture containing a pywemo class name used by pywemo_device_fixture."""
+    return "Insight"
+
+
+@pytest.fixture(name="pywemo_registry")
+def pywemo_registry_fixture():
+    """Fixture for SubscriptionRegistry instances."""
+    registry = asynctest.create_autospec(pywemo.SubscriptionRegistry)
```

Comment:
Please don't use asynctest directly. Import all mock tools from `tests.async_mock`, ie import `create_autospec` from that module. That module is used for Python version compatibility.